CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Added
+
+- Add Instrumentation Scope and Version as info metric and label in Prometheus exporter. (#3273)
```

Comment:
Probably also good to mention the new option if people want to disable Instrumentation scope setting